.ci/scripts/test.sh

```diff
@@ -127,12 +134,6 @@ for link_mode in ${link_modes}; do
             export CC=icx
             export CXX=icpx
         fi
-        echo "============== Configuration: =============="
```

Comment:
Why this is removed? just printing state of configs

Reply:
Thanks for pointing out @napetrov. 
This is removed unintentionally, will revert changes.